.gitignore

```diff
@@ -23,6 +23,8 @@ schnorr_example
 *.trs
 *.sage.py
 
+.vscode/
+
```

Comment:
nit: I think this is something you should set on your local machine, either just for this repo, or for all repos, see https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer for instructions for both variants.